story.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Carpool Vote</title>
```

Comment:
Can we please change the title to something like 'Our Story | Carpool Vote'? What do you reckon?